File: crates/fresh-editor/tests/fixtures/syntax_highlighting/hello.clj
;; Clojure syntax highlighting test
(ns hello.core)

(defn greet [name]
  (str "Hello, " name "!"))

(def config {:version "1.0"
             :enabled true
             :count 42})

(defn -main [& args]
  (println (greet "World"))
  (doseq [item [1 2 3]]
    (println "Item:" item)))

File: crates/fresh-editor/tests/fixtures/syntax_highlighting/hello.fs
// F# syntax highlighting test
module Hello

let greet name =
    sprintf "Hello, %s!" name

type Config = {
    Version: string
    Enabled: bool
    Count: int
}

let config = {
    Version = "1.0"
    Enabled = true
    Count = 42
}

[<EntryPoint>]
let main argv =
    let message = greet "World"
    printfn "%s" message
    0
